### PR TITLE
Remove unnecessary ignores

### PR DESCRIPTION
**Pull request by alex-semenyuk**
Remove unnecessary ignores. Currently it seems as expected result

### CODE REVIEW

File: src/test/java/org/cactoos/map/NoNullsTest.java [hunk 1]
```diff
@@ -297,8 +295,7 @@ public void removeWithNullKey() {
         );
     }
 
-    @Test
-    @Ignore
+    @Test(expected = IllegalStateException.class)
```
Comment (alex-semenyuk) [inline src/test/java/org/cactoos/map/NoNullsTest.java:298]:
```
        final V result = this.map.remove(key);
        if (result == null) {
            throw new IllegalStateException(
                new UncheckedText(
                    new FormattedText(
                        "Value returned by #remove(%s) is NULL",
                        key
                    )
                ).asString()
            );
        }
```

File: src/test/java/org/cactoos/map/NoNullsTest.java [hunk 2]
```diff
@@ -251,8 +250,7 @@ public void putWithNullValue() {
...
```
Comment (alex-semenyuk) [inline src/test/java/org/cactoos/map/NoNullsTest.java:253]: final V result = this.map.put(key, value);
        if (result == null) {
            throw new IllegalStateException(
                new UncheckedText(
                    new FormattedText(
                        "Value returned by #put(%s,%s) is NULL",
                        key, value
                    )
                ).asString()
            );
        }

**Comment by victornoel** [comment]
Hi @alex-semenyuk, I'm not exactly sure what are the meaning of the comments you added… can you enlighten me?

Thank you for the PR :)

**Comment by alex-semenyuk** [comment]
Just to show from where the result come from and that it's expected

**Pull request closed without merging**